SupportClient/SupportClient/Views/AddImageView.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  AddImageView.swift
+//  SupportClient
+//
+//  Created by Cameron Russell on 3/11/20.
+//  Copyright © 2020 Cornell Appdev. All rights reserved.
+//
+
+import UIKit
+
+class AddImageView: UIView {
+
+    private let addFileButton = UIButton()
+    private let noFileLabel = UILabel()
+    private var onPress: (() -> Void)?
+    
+    override var intrinsicContentSize: CGSize {
+        return UIScreen.main.bounds.size
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    init(onPress: @escaping () -> Void) {
```

Comment:
does this have to be `@escaping`?

Reply:
`onPress` is an escaping closure according to the error message when I remove it, so yes